addon/src/webextension/lib/metrics.js

```diff
@@ -1,148 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
bootstrap-removal, daily-ping was written fresh using browser.alarms in background.js